README.md

```diff
@@ -748,16 +764,13 @@ The root finder can be called with.
 const epsilon = 0.001;
 const finder = new module.NewtonRaphson(epsilon);
 const guess = -20;
-const root = finder.find(guess);
+const root = finder.solve(guess);
 ```
 
 Append the root answer to the html page using [document manipulation functions](https://developer.mozilla.org/en-US/docs/Web/API/ParentNode/append).
```

Comment:
Not a append but a set or update

Reply:
Where does 'answer' come from. Move html page up or explain that it's going to be made